config.go

```diff
@@ -18,11 +18,12 @@ const (
 var (
 	defaultHomeDir = dcrutil.AppDataDir("dcrtweetbot", false)
 
-	defaultConfigFile  = filepath.Join(defaultHomeDir, defaultConfigFilename)
-	defaultTargetWords = []string{}
-	defaultIPFSHost    = "localhost:5001"
-	defaultDcrtimeHost = v1.DefaultMainnetTimeHost
-	defaultDcrtimePort = v1.DefaultMainnetTimePort
+	defaultConfigFile    = filepath.Join(defaultHomeDir, defaultConfigFilename)
+	defaultTargetWords   = []string{}
+	defaultIPFSHost      = "localhost:5001"
+	defaultDcrtimeHost   = v1.DefaultMainnetTimeHost
+	defaultDcrtimePort   = v1.DefaultMainnetTimePort
+	defaultEnableReplies = false
```

Comment:
Should it by true instead ? :thinking:

Reply:
I like it as false. It's weird that this is enabled by default. Especially because depending on the target word you use you can easily consume all your twitter API requests for the day.